audio/piper-phonemize/files/piper_phonemize.pc

```diff
@@ -0,0 +1,11 @@
+prefix=/usr/local
```

Comment:
Use `prefix=%%PREFIX%%` there and then use `${SED} -e 's|%%PREFIX%%|${PREFIX}|' ...` to install this file in `post-stage`.